www/source/blog/2018-10-12-shelving-composites.html.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Shelving Composites (for now)
+date: 2018-10-12
+author: Garrett Amini
+tags: supervisor, packaging
+category: supervisor
+classes: body-article
+---
+
+Nearly exactly one year ago today, Christopher Maier [presented the ability to create _composite packages_](https://www.habitat.sh/blog/2017/10/Introducing-Composites/), a special kind of Habitat package that includes other Habitat packages in order to be managed by a single Supervisor. The Habitat community had been finding all sorts of creative ways to use Habitat to incorporate their services under a single Supervisor, so we sought a way to canonically enable them to do so. 
+
+This past year has taught us that while we were on the right track with the idea of composite packages, our implementation left much to be desired. Users kept trying to utilize composite packages, but would inevitably find themselves bumping up against limitations of the feature, and would thus end implementing the same sorts of creative solutions we were trying to simplify in the first place.
+
+## What's Happening Now?
+
+Starting with version *X.XX.X*, we will be removing the ability to run composite plans from Habitat. We've learned enough over the past year to realize that we need a complete re-implementation of how that feature will work with the Supervisor, so we don't want to lead our users down a dead-end path. This is one of those cases where rebuilding the feature will be much faster to implement than refactoring what we have, so we are taking the hard step of removing the current implementation now.
+
+## The Path Forward
+
+As we near 1.0, we are seeking to make Habitat as robust and reliable as possible. Once we achieve that milestone, we're looking forward to once again adding the ability to create composite packages, developed with help and feedback from the community.

```

Comment:
It may be that we don't actually have a different kind of package, but provide ways of limiting binds to the same supervisor, rather than going out over the network.

So (without saying that in the post), I'd wouldn't say that we're going to "re-implement composite packages", but rather that "we'll be looking at how to satisfy the usecases that composite packages were meant to serve" (or words to that effect).

And definitely with feedback from the community. 👍